Upstream-Status: Backport
Signed-off-by: Ross Burton <ross.burton@arm.com>

From ca6c179226864bff23f2b062518cf885bb42ce56 Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Thu, 27 Apr 2023 15:26:37 +0200
Subject: [PATCH] stdlib: Fix error when cross-compiling.

Reported by Pierre Labastie <pierre.labastie@neuf.fr> in
<https://lists.gnu.org/archive/html/bug-gnulib/2023-04/msg00220.html>.

* m4/stdlib_h.m4 (gl_STDLIB_H): Provide a 4th argument to AC_RUN_IFELSE.
---
 ChangeLog      | 7 +++++++
 m4/stdlib_h.m4 | 4 ++--
 2 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/m4/stdlib_h.m4 b/m4/stdlib_h.m4
index 3274ea4948..f47c1eb37b 100644
--- a/m4/stdlib_h.m4
+++ b/m4/stdlib_h.m4
@@ -66,7 +66,7 @@ int main ()
   return result;
 }]])],
           [gl_cv_macro_MB_CUR_MAX_good=yes],
-          [gl_cv_macro_MB_CUR_MAX_good=no]
+          [gl_cv_macro_MB_CUR_MAX_good=no],
           [:])
       fi
     ])
-- 
2.34.1

